stories/dynamic-page/dynamic-page.stories.js

```diff
@@ -0,0 +1,929 @@
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+                Dolor sit Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eius assumenda, quidem natus optio repudiandae deleniti 
+                in atque quis, sed cum asperiores minus rerum incidunt unde quod fuga amet ea reprehenderit.
+            </div>
+        </div>
+        <div class="fd-bar fd-bar--floating-footer">
+            <div class="fd-bar__right">
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--emphasized fd-button--compact">Save</button>
+                </div>
+                <div class="fd-bar__element">
+                    <button aria-label="button" class="fd-button fd-button--transparent fd-button--compact">Cancel</button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <br /><br />
+`;
+
+desktop.storyName = 'Desktop mode';
+desktop.parameters = {
+    docs: {
+        iframeHeight: 150,
+        storyDescription: 'The dynamic page with Desktop specification(L). The same specification also applies to XL and XXL as well.' +
+        'Please note the difference in the way tabs are used. Tabs must use `fd-dynamic-page__tabs` along with the size modifiers(--m, --l, --xl or --s) and' +
```

Comment:
What are the directions for the size modifiers? Does the size depend on how many tabs you have open?

**About tabs**

- Seeing as this seems like an optional component (I checked out Fiori 3 and there is no mention of tabs) and the Visual Core wiki only says:

![Screen Shot 2020-09-17 at 12 55 52 PM](https://user-images.githubusercontent.com/57453945/93502737-20389600-f8e5-11ea-8376-738c6501741f.png)
![Screen Shot 2020-09-17 at 12 56 41 PM](https://user-images.githubusercontent.com/57453945/93502817-3e9e9180-f8e5-11ea-8e9d-55d0e10a4c08.png) 

- I think we should mention that tabs are being used optionally here.
- Also it would be good to link **Tabs** it to its page when we mention it for the first time, otherwise you would write it like you have been: "tabs".

Reply:
> Does the size depend on how many tabs you have open?

No, it isn't dependent on the opened tabs. It is to do with the paddings of the tab container, much like the ones being applied for `title-area`, `header` etc.
I have added the tabs modifier as another row in the desktop/tablet/mobile mode tables.

> I think we should mention that tabs are being used optionally here.

Have added the line: 
"**Tabs** can be optionally added as an extension of the header area with the tab content being a part of the page content." Is this okay?